keycloak/keycloak-operator
# remove extra rule for git.apache.org

Remove the extra rule that was required for go < 1.12

Verification steps:

1. Make sure that `make code/compile` and `make test/unit` still work with this branch
